src/components/CouponDetails/index.jsx

```diff
@@ -41,16 +41,13 @@ class CouponDetails extends React.Component {
     ];
 
     this.state = {
-      selectedFilter: null,
+      selectedToggle: 'not-assigned',
```

Comment:
[ENT-1345](https://openedx.atlassian.net/browse/ENT-1345) mentions to default filter (toggle now) to fetch all coupon codes records. Here we will be defaulting to 'not-assigned'?

Reply:
In recent discussions with UX and Product, we decided to treat this as toggles instead of filters, where the initial table data will be unassigned codes. I will reconfirm this with UX/Product today, though.

Reply:
@mushtaqak Chatted with UX about this again to confirm that initial table data will be only unassigned codes.